graphql_api/types/owner/owner.graphql

```diff
@@ -20,6 +20,7 @@ type Owner {
   ownerid: Int!
   plan: Plan
   pretrialPlan: PlanRepresentation
+  availablePlans: [PlanRepresentation!]!
```

Comment:
What's the difference between `[PlanRepresentation!]!` and `[PlanRepresentation]!`?